src/main/java/org/datadog/jmxfetch/JMXAttribute.java

```diff
@@ -355,4 +355,8 @@ String getBeanName() {
     String getAttributeName() {
         return attributeName;
     }
+
+    public static List<String> getExcludeBeanParams(){
```

Comment:
Just to be consistent, could we rename this method `getExcludedBeanParams`? :)  (ie `Exclude` --> `Excluded`)